index.html

```diff
@@ -38,12 +37,191 @@ <h1 class="greetings">
+          <h3 class="section-title margin-top-12">to do list app</h3>
+          <div class="work-period">
+            <ul>
+              <li class="text-color-primary">Canopy</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p class="work-description text-color-primary margin-top-20">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="work-categories margin-top-12 padding-0">
+            <li><span>html</span></li>
+            <li><span>css</span></li>
+            <li><span>javascript</span></li>
+          </ul>
+          <div>
+          <button type="button" class="button button-enabled">
+            See Project
+          </button>
+          </div>
+        </div>
+        <div class="flex works-grid-item">
+          <img
+            class="width-100"
+            src="image/fitness.png"
+            alt="fitness screenshot"
+          />
+          <h3 class="section-title margin-top-12">fitness</h3>
+          <div class="work-period">
+            <ul>
+              <li class="text-color-primary">Canopy</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p class="work-description text-color-primary margin-top-20">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="work-categories margin-top-12 padding-0">
+            <li><span>html</span></li>
+            <li><span>css</span></li>
+            <li><span>javascript</span></li>
+          </ul>
+          <div>
+          <button type="button" class="button button-enabled">
+            See Project
+          </button>
+          </div>
+        </div>
+        <div class="flex works-grid-item">
+          <img
+            class="width-100"
+            src="image/styling.png"
+            alt="styling screenshot"
+          />
+          <h3 class="section-title margin-top-12">printing</h3>
+          <div class="work-period">
+            <ul>
+              <li class="text-color-primary">Canopy</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p class="work-description text-color-primary margin-top-20">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="work-categories margin-top-12 padding-0">
+            <li><span>html</span></li>
+            <li><span>css</span></li>
+            <li><span>javascript</span></li>
+          </ul>
+          <div>
+          <button type="button" class="button button-enabled">
+            See Project
+          </button>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="about-myself padding-top-114">
```

Comment:
- [ ] please use the  correct background color for this section and the heading and button should be well aligned.
your design
![Screenshot (64)_LI](https://user-images.githubusercontent.com/65298048/170498653-e90b980a-b062-46cb-84d5-75dead76ab19.jpg)
- figma design 

![Screenshot (63)](https://user-images.githubusercontent.com/65298048/170498923-d6faa8f5-b3e1-47f4-b566-fa04b83d2660.png)